### PR TITLE
Milestone 1:Setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by pasytchangwa**
## Create toolbar and headline section for my portfolio project

The following was done :

- Setup files configurated
- Header was added and styled
- Headline section added and styled
- linters verified

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Home</title>
+  </head>
+  <body>
+   <div class="wrapper">
+     <header class="toolbar">
+       <div class="logo">
+         <img src="./image/images-head/logo.png" alt="MyLogo" />
```
Comment (mricanho) [inline index.html]: - [ ] This logo it's not only an image but a link, this is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+       </div>
+       <nav class="nav-btn">
+         <img src="./image/images-head/union.png" alt="navbarimage" />
```
Comment (mricanho) [inline index.html:16]: - [ ] Please align your hamburger with the logo, check your design:
image
 and the original:
image

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @pasytchangwa!

Good job so far! 🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

almost there

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (pasytchangwa) [reply index.html]: Thank you for the recommendation. link added for it to be directed to the home page.

File: index.html (hunk 2 above)
Comment (pasytchangwa) [reply index.html:16]: Hamburger aligned to the logo as requested.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @pasytchangwa🔥 

You worked hard in all the instructions that I give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**